### PR TITLE
php artisan cache:show [store]

### PR DESCRIPTION
**Pull request by jordanbardsley7**
I think this would be a super cool addition to the artisan console!

screenshot 2016-09-01 14 23 52

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@ composer.lock
 .php_cs.cache
 .DS_Store
 Thumbs.db
-/phpunit.xml
\ No newline at end of file
+/phpunit.xml
+/.idea
```
Comment (GrahamCampbell) [inline .gitignore]: Please don't add this.

**Comment by GrahamCampbell** [comment]
Thanks for the PR. Some comments: This doesn't allow you to select the connection, and it doesn't feel right to have to tell us what the driver is. We can trivially work that out.

**Comment by jordanbardsley7** [comment]
@GrahamCampbell Hey Graham, i'm not too sure how the phpunit.xml got into the gitignore I will have that removed. What sort of recommendations do you have to get this merged?

**Comment by taylorotwell** [comment]
No plans to add this as many caches would have way too many keys to reasonably show.

**Comment by jordanbardsley7** [comment]
@taylorotwell Could it not have any benefit for something maybe like cache:find or limiting the most recent cache items?

**Pull request closed without merging**